docs.html

```diff
@@ -10,10 +10,26 @@
   <body>
     <div class="site">
       <header class="site-header">
-        <div class="site-name">UI Library</div>
+        <div class="site-header-title">Adaptive UI</div>
+        <div class="site-header-ctas">
+          <a href="#">
+            <i class="fab fa-github"></i>
+          </a>
+          <a href="#">
+            <i class="fab fa-linkedin"></i>
+          </a>
+          <a href="#">
+            <i class="fab fa-twitter"></i>
+          </a>
+        </div>
```

Comment:
this is good!!

Reply:
this is looking good I like the naming convention